### PR TITLE
Add compat data for nullish coalescing operator

### PR DESCRIPTION
**Pull request by SphinxKnight**
This adds compat data for ECMAScript "nullish coalescing operator". This feature is implemented in 2 browsers which are nightly/previews. The TC39 proposal is stage 3.

* [TC39 Proposal](https://github.com/tc39/proposal-nullish-coalescing)
* [SpiderMonkey/Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1566141) (landed in Nightly yesterday/during the night) 72.0a1 (2019-11-12) (tested: working)
* [Safari Release notes](https://webkit.org/blog/9497/release-notes-for-safari-technology-preview-89/): Safari Technology Preview 89
* [Chromium bug](https://bugs.chromium.org/p/v8/issues/detail?id=9547) / [intent to ship](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/7zLAOqxWxbE/IFifZv0gEAAJ) 
* [Test262 Report](https://test262.report/browse/language/expressions/coalesce)
* [MDN page](https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)

So basically all browsers are at `false` right now but the feature is on pretty standard tracks and implemented in two preview/nightly browsers.

### CODE REVIEW

File: javascript/operators/nullish_coalescing_operator.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+  "javascript": {
+    "operators": {
+      "optional_chaining": {
```
Comment (Elchi3) [inline javascript/operators/nullish_coalescing_operator.json]:
```suggestion
      "nullish_coalescing": {
```

File: javascript/operators/nullish_coalescing_operator.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        "__compat": {
+          "description": "Nullish coalescing operator (<code>??</code>)",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator",
+          "spec_url": "https://tc39.es/proposal-nullish-coalescing/#top",
```
Comment (Elchi3) [inline javascript/operators/nullish_coalescing_operator.json]: Maybe this?

```suggestion
          "spec_url": "https://tc39.es/proposal-nullish-coalescing/#sec-conditional-operator",
```

Comment (SphinxKnight) [reply javascript/operators/nullish_coalescing_operator.json]: Well, I'm not sure that pointing to the grammar of the ternary operator is helpful.
By default, I'd suggest https://tc39.es/proposal-nullish-coalescing/#sec-intro instead.

I don't know if this topic is tracked elsewhere but when submitting, having to put a fragment to validate the schema seemed weird to me (at least for this case where a proposal targets a single feature / I reckon it's helpful in a longer/richer spec).

Comment (Elchi3) [reply javascript/operators/nullish_coalescing_operator.json]: Yeah, we're discussing currently whether requiring fragments make sense. At the point this lands in the main spec, it does make sense to have a good fragment id, but right now it doesn't really.

Comment (Elchi3) [reply javascript/operators/nullish_coalescing_operator.json]: #top works for now. Thank you!

**Comment by Elchi3** [comment]
I just looked in the folder and we haven't added "_operator" to the other files. What do you think about changing 
`javascript/operators/nullish_coalescing_operator.json`
to
`javascript/operators/nullish_coalescing.json` ?
